index.html

```diff
@@ -70,5 +70,314 @@ <h1 class="headline_title">
         </ul>
       </div>
     </section>
-  </body>
-</html>
+    <section>
+        <div>
+         <div>
+             <h1 class="batu">
+                 My Recent <br> Works
+             </h1><hr>
+         </div>
+        <div class="imgplacesholder"></div>
+             <div>
+                 <h2 class="engin">
+                     Multi-Post Stories <br> Gain+Glory
+                 </h2>
+                 <ul class="eylem">
+                     <li class="irem">
+                         <button class="btn1">
+                             Ruby on rails
+                         </button>
+                     </li>
+                     <li class="irem">
+                         <button class="btn">
+                             css
+                         </button>
+                     </li>
+                     <li class="irem">
+                         <button class="btn1">
+                             JavScript
+                         </button>
+                     </li>
+                     <li class="irem">
+                         <button class="btn1">
+                             html
+                         </button>
+                     </li>
+                 </ul>
+                 <button class="btngreen">
+                     See Project
+                 </button>
+             </div>
+         </div>
+
+         <div class="imgplacesholder"></div>
+             <div>
+                 <h2 class="engin">
+                     Multi-Post Stories <br> Gain+Glory
+                 </h2>
+                 <ul class="eylem">
+                     <li class="irem">
+                         <button class="btn1">
+                             Ruby on rails
+                         </button>
+                     </li>
+                     <li class="irem">
+                         <button class="btn">
+                             css
+                         </button>
+                     </li>
+                     <li class="irem">
+                         <button class="btn1">
+                             JavScript
+                         </button>
+                     </li>
+                     <li class="irem">
+                         <button class="btn1">
+                             html
+                         </button>
+                     </li>
+                 </ul>
+                 <button class="btngreen">
+                     See Project
+                 </button>
+             </div>
+         </div>
```

Comment:
Kindly note the following changes you should make to the cards section as regards styling and CSS:

- [ ] The cards have a `margin` of `26px` and `24px` to the left and right sides respectively. They also have a `margin-bottom` of `24px.` Kindly design it that way. 💻 
- [ ] There is a border of `border: 1px solid #D0D9D4` and a border radius of `border-radius: 8px` on each card/project. Kindly implement it so that each card is identical to the cards of the Figma template that you chose. 👍 

### Your Design

![image](https://user-images.githubusercontent.com/77106962/205452146-9e0e0924-9f16-4e3a-a152-3b6963a07526.png)

### Figma Design

![image](https://user-images.githubusercontent.com/77106962/205452180-4b9a4195-462d-4825-b48e-fa7460a1c5e7.png)

#### Tips
- I would advise you to put the `div` with class `imgplacesholder` in the general div of the cards below it. 
- Then apply the `margins` I recommended above and
- Apply the `border` declarations also. 💻